Control/Monad/Logger.hs

```diff
@@ -380,6 +383,11 @@ instance MonadTransControl NoLoggingT where
     {-# INLINE liftWith #-}
     {-# INLINE restoreT #-}
 
+#if __GLASGOW_HASKELL__ >= 801
+instance (Fail.MonadFail m) => Fail.MonadFail (NoLoggingT m) where
```

Comment:
For each instance, please add a Haddock including a `@since` comment. Also, please perform a PVP minor bump in the cabal file and add a mention of the new instances in the ChangeLog.

Reply:
Fixed using rebase. Thanks for the guidance.